ADMIN.adoc

```diff
@@ -0,0 +1,35 @@
+= Repository Administrator Responsibilities
+
+== Administrator rotation schedule:
+
+Every 2 weeks a different project member assumes the role of repository administrator.
+Signing up to be administrator is voluntary.
+
+== Administrator responsibilities:
+
+As the administrator, you are in charge of performing the following duties:
+
+=== Being the Point of Contact for general issues related to the Supplementary Style Guide repository
+
+* You are the point of contact in case other contributors experience issues with the repository.
+When people reach out to you about an issue, try to fix it. If you cannot fix the issue, report it to a member of the initiative who can.
+
+=== Maintaining the Style Guide site on GitHub
```

Comment:
I would also recommend updating the instruction to flag the issue with "Needs Fix" (we can add that if people agree) and simply @ the admin group (redhat-documentation/ccs-style-council)

Reply:
> @inoxx03 Thanks a lot for drafting these responsibilities! Just a few minor comments.
> 
> What do you think about including a section to this page from the README page? Otherwise people might not know that it exists.

@bergerhoffer I think that is a good point. Let me add the link after I incorporate the collective feedback form project members. Thank you.